components/LocationDetails/index.js

```diff
@@ -39,46 +40,128 @@ const StyledDiv = styled.div`
       <StyledArticle>
         <NavLink href="/locations"> ← Back</NavLink>
         <StyledDiv>
-          <p>{addresse}</p>
-          <p>{zeit}</p>
+          <p>{data?.location}</p>
+          <p>{data?.zeit}</p>
           <StyledArtSection>
-            {art?.map((artStück) =>
+            {data?.art?.map((artStück) =>
               artStück === "Cafe" ? (
-                <Image key={1} src="/images/CafeIcon.svg" width={71} height={45} alt="Cafe icon"/>
+                <Image
+                  key={1}
+                  src="/images/CafeIcon.svg"
+                  width={71}
+                  height={45}
+                  alt="Cafe icon"
+                />
               ) : artStück === "Restaurant" ? (
-                <Image key={2} src="/images/RestaurantIcon.svg" width={71} height={44} alt="Restaurant-icon"/>
+                <Image
+                  key={2}
+                  src="/images/RestaurantIcon.svg"
+                  width={71}
+                  height={44}
+                  alt="Restaurant-icon"
+                />
               ) : artStück === "Bar" ? (
-                <Image key={3} src="/images/BarIcon.svg" width={57} height={44} alt="Bar-icon"/>
+                <Image
+                  key={3}
+                  src="/images/BarIcon.svg"
+                  width={57}
+                  height={44}
+                  alt="Bar-icon"
+                />
               ) : artStück === "Kuchen" ? (
-                <Image key={4} src="/images/KuchenIcon.svg" width={71} height={44} alt="Kuchen-icon"/>
+                <Image
+                  key={4}
+                  src="/images/KuchenIcon.svg"
+                  width={71}
+                  height={44}
+                  alt="Kuchen-icon"
+                />
               ) : artStück === "Eis" ? (
-                <Image key={5} src="/images/EisIcon.svg" width={57} height={45} alt="Eis-icon"/>
+                <Image
+                  key={5}
+                  src="/images/EisIcon.svg"
+                  width={57}
+                  height={45}
+                  alt="Eis-icon"
+                />
               ) : artStück === "Snacks" ? (
-                <Image key={6} src="/images/SnacksIcon.svg" width={71} height={44} alt="Snacks-icon"/>
+                <Image
+                  key={6}
+                  src="/images/SnacksIcon.svg"
+                  width={71}
+                  height={44}
+                  alt="Snacks-icon"
+                />
               ) : (
                 []
               )
             )}
           </StyledArtSection>
-          <p>{verleih}</p>
-          <Image src={bild} height={62} width={350} alt={name} />
+          <p>{data?.verleih}</p>
+          <Image
+            src={data?.bild.img}
+            height={62}
+            width={350}
+            alt={data?.name}
+          />
         </StyledDiv>
+        <NotesForm locData={data} onSubmit={handleSubmit} />
+        <StyledUl>
```

Comment:
We might run into a situation where `data.notes` is an empty array. We would then end up with some html as follows

``` html
<ul>
</ul>
```

Maybe it only makes sense to have the `ul` if there are some notes. 

i.e.

``` js
{data?.notes.length > 0 && ...}


```

Reply:
That's a good idea, thanks!